pkg/k8s/apps/crud.go

```diff
@@ -50,6 +50,41 @@ func Get(ctx context.Context, dev *model.Dev, namespace string, c kubernetes.Int
 	return &StatefulSetApp{sfs: sfs}, nil
 }
 
+//IsDevModeOn returns if a statefulset is in devmode
+func IsDevModeOn(app App) bool {
+	return app.ObjectMeta().Labels[model.DevLabel] == "true"
+}
+
+func (t *Translation) DevModeOff() {
+	t.App.DevModeOff(t)
```

Comment:
This generates a weird circular dependency. If `App.DevModeOff` would accidentally call to `t.DevModeOff()`, we would have an infinite loop. Is there a way we could avoid it? Would it make more complex the code?

Reply:
I don't like it either :-)
But next refactor will eliminate the object `Translation` as it is, I think it is better to fix it on that iteration

Reply:
Sounds good!